_includes/extensions/writing-repeater.html

```diff
@@ -27,158 +47,131 @@ <h2 id="writing-extensions-repeater">Repeater Views</h2>
-			</td>
+			<td>Runs every time this repeater view is selected prior to rendering. Provided a helpers object containing
+				the previous repeater view (<code>prevView</code>)</td>
 		</tr>
 		<tr>
 			<td>before</td>
 			<td>function</td>
-			<td>Runs before the <code>render</code> function. Provided a <a href="#writing-extensions-repeater-helpers">helpers</a>
-				object and <code>callback</code> function. The callback accepts an object with the <code>skipNested</code>
-				attribute. If the <code>skipNested</code> value is <code>true</code>, the repeater will ignore
-				rendering the <code>nested</code> array for this renderer.
+			<td>This function is called prior to the <code>renderItems</code> function, if no <code>render</code> function has been defined.
+				Provided a <code>helpers</code> object containing the parent <code>container</code> (the repeater canvas)
+				for items to be inserted and <code>data</code> returned from the dataSource. Expects to be returned an
+				<a href="#writing-extensions-repeater-itemObject">item object</a>, or <code>false</code> if no action is desired.
 			</td>
 		</tr>
 		<tr>
-			<td>complete</td>
+			<td>renderItem</td>
 			<td>function</td>
-			<td>Runs after all the renderers in the <code>nested</code> array have ran. Provided a
-				<a href="#writing-extensions-repeater-helpers">helpers</a> object and <code>callback</code>
-				function.
+			<td>This function is called after the <code>before</code> function, if no <code>render</code> function has been defined.
+				It is iterated over the array specified by the <code>repeat</code> attribute. It is provided a <code>helpers</code>
+				object containing the parent <code>container</code> (which will be whatever element within the repeater canvas
+				has the <code>data-container="true"</code> attribute, or the canvas itself if not found) for items to be inserted,
+				the <code>data</code> returned from the dataSource, the <code>index</code> of the current item within the iteration,
+				and the <code>subset</code> array of data that the renderItem function is being iterated on. Expects to be returned an
+				<a href="#writing-extensions-repeater-itemObject">item object</a>, or <code>false</code> if no action is desired.
 			</td>
 		</tr>
 		<tr>
-			<td>nested</td>
-			<td>array</td>
-			<td>Array of renderer objects. These objects use the same structure as the one currently being described.
-				Each object will run in order following the <code>render</code> function of the current object unless
-				otherwise specified.
+			<td>repeat</td>
+			<td>string</td>
+			<td>This string value specifies which array the <code>renderItem</code> function iterates over. It represents
+				an attribute of an object using dot notation. The root level value must use <code>'data'</code> or
+				<code>'this'</code>. If the root value uses <code>'data'</code>, the root object will be the <code>data</code>
```

Comment:
One is a string, while the other is a variable. Basically, it's saying the string `'this.something.anotherThing'` will be mapped to `this.something.anotherThing`